Engineering/Management/Story point estimation.md

```diff
@@ -0,0 +1,26 @@
+---
+tags: engineering/management, team-topologies
+author: Nhut Huynh
+date: 2022-09-15
+---
+
+During the team sprint-planning meeting, story points estimation is one of the important part when defining the works in the future sprint for each members in the team. A good estimation will help product owners optimize for efficiency and impact. How do we do the estimation precisely?
+
+## Collaborate with your project manager, product owner
+Usually, before the sprint planning happens, PM/PO will take time to prepare for the tickets are going to do in the future sprint - most of the tickets are captured based on the desired features and fixs for a product from customer's requirements.
+ 
+Hence, those tickets are in the very common statement since most of the PM/PO does not understand much from the technical site for the implementation. Leading to hard for all of the members to estimate their work.
+
+So, to separate works precisely and more easy to estimate, a team members can give PMs new insight into the level of effort for each work item. Like, can giving PMs a definition of their work for example.
+
+## Agile estimation is a team sport
+During the estimation, it should be involving by eveyone members in the team (developers, designers, testers, etc.). Why? Because each of them will have a different perspective on the product and the work required to deliver for the feature.
+
+## Estimate smarter, not harder
+Usually, when estimating, we will based on the time that our member need to finish their work. For example: 1-2 points for about 2-4 hours. So when having a big requirement ticket, we should break it into the subtasks or smaller tasks to keep track the work better and easy for the team to estimate also.
```

Comment:
Should mentions to Fibonacci sequence value and using planning poker when doing estimation.